litellm/litellm_core_utils/litellm_logging.py

```diff
@@ -900,6 +918,43 @@ def success_handler(
                             end_time=end_time,
                             print_verbose=print_verbose,
                         )
+                    if callback == "literalai":
```

Comment:
same comment here - this should be avoided. 

as long as this is a CustomLogger class, we know how to log to it. 

https://github.com/BerriAI/litellm/blob/ab153c3e8dab6be093eba3726ada19da176ee8cc/litellm/litellm_core_utils/litellm_logging.py#L2137